docs/our_work/ds218_rap_community_of_practice.md

```diff
@@ -1,29 +1,42 @@
 ---
 layout: base
 title: NHS Reproducible Analytical Pipelines (RAP) Champion Squad
-permalink: ds218_rap_community_of_practice.html
+summary: Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also taught them how to train others and produced guidance which is used by many other organisations.
+origin: 'NHS Digital'
+tags: ['RAP','Best Practice','Python', 'R']
```

Comment:
can we make all of the tags capitalized? (unless you decide that my makign them consistently capitalized in my PR is a bad move)

Reply:
I'll make these changes :)